src/main/java/net/glowstone/generator/decorators/overworld/IceDecorator.java

```diff
@@ -26,7 +26,7 @@ public void populate(World world, Random random, Chunk chunk) {
                 y--;
             }
             //if (world.getBlockAt(x, y, z).getType() == Material.SNOW_BLOCK) {
```

Comment:
Was IceDecorator done before we added snow to the highest block in the appropriate biomes?